minishell.h

```diff
@@ -36,6 +36,9 @@
 
 # define PROMPT "\e[0;35mminishell$ \e[0m"
 
+extern int	g_last_exit_code;
```

Comment:
I am unsure how to understand this from the subject:

> Avoid using more than one global variable to indicate a received signal. Consider the implications: this approach ensures that your signal handler will not access your main data structures.

Is this the variable to store `received_signal`? if yes, let's rename, if not let's discuss why do we need this one here